rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+version = "1.75.0"
```

Comment:
A good resource: https://gribnau.dev/cargo-msrv/commands/find.html

Reply:
The toolchain is what the compiler and tooling for this workspace will default too. Is there any requirement to support older versions? Using older versions can give difficulty later in upgrading libraries. Unless there's some specific requirement to support older versions we should go ahead with the latest and greatest. Especially so because we plan to pull in the tokio and async ecosystem which is still evolving.